network.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface TablelandNetworkConfig {
```

Comment:
Now that I think about it, since these are generated files, is there a reason we have these in VC? Or should we just be generating them as part of the build and release process?

Reply:
hmm, good point. lookin...

Reply:
seems easy enough just gitignore those files. related question, should we create a unified index of the exports? so that you don't have to import form `@tableland/evm/network.js` ? Here's what the package looks like:

<img width="252" alt="Screen Shot 2022-10-10 at 1 00 15 PM" src="https://user-images.githubusercontent.com/361000/194943714-0fcda5eb-0f92-4956-84da-ae27cd960496.png">

and the current export pattern: 

<img width="295" alt="Screen Shot 2022-10-10 at 1 01 17 PM" src="https://user-images.githubusercontent.com/361000/194943683-3876e246-6974-42df-ae4b-ec58adb03243.png">